person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  reader_attr :id
+  reader_attr :id, :rentals
   accessor_attr :name, :age
```

Comment:
- [ ] Please change this to `attr_accessor`
- [ ] Please add rentals in the `attr_accessor`